tools/GenerateTestSummary/TestSummaryGenerator.cs

```diff
@@ -66,7 +66,7 @@ public static string CreateCombinedTestSummaryReport(string basePath)
                             ? "lin"
                             : filePath.Contains("macos-")
                                 ? "mac"
-                                : "os?";
+                                : throw new InvalidOperationException($"Could not determine OS from file path: {filePath}");
 
             tableBuilder.AppendLine(CultureInfo.InvariantCulture, $"| {(failed > 0 ? "❌" : "✅")} [{os}] {GetTestTitle(filePath)} | {passed} | {failed} | {skipped} | {total} |");
```

Comment:
I noticed in my tests that even when there are not test results (i.e., all 0, can happen when the runner crashed) we show the "tick". It looks odd.

Reply:
Can you share an example of that, so I can fix accordingly?

Reply:
There you go: https://github.com/RussKie/aspire/actions/runs/14984406738
![image](https://github.com/user-attachments/assets/9322fb83-e070-4759-b233-bdefb17b43ad)